addons/onboarding/models/onboarding_progress.py

```diff
@@ -23,9 +23,20 @@ class OnboardingProgress(models.Model):
     onboarding_id = fields.Many2one(
         'onboarding.onboarding', 'Related onboarding tracked', required=True, ondelete='cascade')
     progress_step_ids = fields.One2many('onboarding.progress.step', 'progress_id', 'Progress Steps Trackers')
+    not_per_company = fields.Selection(
+        [('True', 'True')], string='Only one', compute='_compute_not_per_company', store=True,
+        help="Should there only be one record for this onboarding on this database."
+    )
+
```

Comment:
I would document that you are using a Selection instead of a Boolean so that you can have a `null` value in the database and that `UNIQUE` is funny when it comes to `null` in postgres. Also, how future proof (thinking of more recent versions of postgres) it is?